Update use-tcp-endpoint-forward-logs-new-relic.mdx with plugin.type information (#20312)
newrelic/docs-website: src/content/docs/logs/log-api/use-tcp-endpoint-forward-logs-new-relic.mdx

   sudo systemctl restart rsyslog
   ```

> 4. [Check your New Relic account](/docs/logs/log-management/ui-data/explore-your-data-log-analytics) for logs (note logs may be found by searching for the attribute plugin.type:"syslog-newrelic").

Reviewer: ```suggestion
4. [Check your New Relic account](/docs/logs/log-management/ui-data/explore-your-data-log-analytics) for logs You can also find logs by searching for the attribute `plugin.type:"syslog-newrelic"`.
```
